### PR TITLE
fix(legacy): invalid css variable

### PR DESCRIPTION
**Pull request by splincode**
Close https://github.com/taiga-family/taiga-ui/issues/8616

### CODE REVIEW

File: projects/legacy/components/textarea/textarea.style.less [hunk 1]
```diff
@@ -167,7 +167,7 @@
     }
 
     &__extra {
-        background-color: var(--tui-status-negative-pale-night);
+        background-color: var(--tui-status-negative-pale);
```
Comment (waterplea) [inline projects/legacy/components/textarea/textarea.style.less:170]: Looks like there's a few other `-night` issues, please search through the code and fix them all while you're at it

Comment (splincode) [reply projects/legacy/components/textarea/textarea.style.less:170]: Can I create another PR?

Comment (waterplea) [reply projects/legacy/components/textarea/textarea.style.less:170]: Why? It's just removing `-night` in 15 places instead of one

Comment (splincode) [reply projects/legacy/components/textarea/textarea.style.less:170]: fixed

**Review by waterplea — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**